server/src/servershot.js

```diff
@@ -326,12 +330,13 @@ class Shot extends AbstractShot {
 
 Shot.getRawBytesForClip = function(uid) {
   return db.select(
-    `SELECT images.url, images.contenttype, data.deviceid
-     FROM images
+    `SELECT images.url, images.contenttype, data.deviceid, devices.accountid
+     FROM images, devices
       JOIN data ON images.shotid = data.id
      WHERE images.id = $1
       AND (data.expire_time IS NULL OR data.expire_time > NOW())
       AND data.block_type = 'none'
+      AND data.deviceid = devices.id
```

Comment:
Query should be fixed to include JOIN on devices something like 
SELECT images.url, images.contenttype, data.deviceid, devices.accountid FROM images 
JOIN data ON images.shotid = data.id
JOIN devices ON data.deviceid = devices.id
WHERE images.id = $1
AND (data.expire_time IS NULL OR data.expire_time > NOW()) 
AND data.block_type = 'none' AND NOT data.deleted

Reply:
Since the condition `data.deviceid = devices.id` is in the WHERE clause, you'll get the same result.  If you swap the placement of `images` and `devices`, that fixes the error.

Reply:
Sorry, my previous comment not specific enough.  Swap `images` and `devices` on line 334 to fix the error.